docker_app/main.py

```diff
@@ -0,0 +1,162 @@
+import re
+import datetime
+import typing
+
+import docker  # type: ignore
+from docker.models import images, containers  # type: ignore
+
+
+class DockerModule:
+    def __init__(self):
+        """
+        Connect to local docker and create docker-client connection
+        """
+        self.client = docker.from_env()
+
+    def close(self) -> None:
+        """
+        Close docker-client connection
+        """
+        self.client.close()
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, exc_type, exc_value, traceback):
+        self.close()
+        if exc_value:
+            raise
+
+    """
```

Comment:
what are these strings in the middle of code? 

Reply:
Additional methods separating